cfme/tests/intelligence/reports/test_tenant_quota_reports.py

```diff
@@ -54,14 +71,49 @@ def test_queue_tenant_quota_reports(appliance, request, set_and_get_tenant_quota
         casecomponent: Reporting
         caseimportance: high
         initialEstimate: 1/10h
+        tags: report
+        setup:
+            1. Set tenant quota
+            2. Create the report
+        testSteps:
+            1. Compare the 'total quota' data in the reports and the quota that was set initially.
+        expectedResults:
+            1. Both the data must be same.
     """
-    tenant_report = appliance.collections.reports.instantiate(
-        type="Tenants", subtype="Tenant Quotas", menu_name="Tenant Quotas"
-    ).queue(wait_for_finish=True)
-    request.addfinalizer(tenant_report.delete)
-
     report_data = dict()
     for row in tenant_report.data.rows:
         if row["Quota Name"] in property_mapping.values():
             report_data[row["Quota Name"]] = row["Total Quota"]
     assert report_data == set_and_get_tenant_quota
+
+
+def test_report_fullscreen_enabled(request, tenant_report, set_and_get_tenant_quota, soft_assert):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: low
+        initialEstimate: 1/12h
+        tags: report
+        setup:
+            1. Navigate to Cloud > Intel > Reports > All Reports
+        testSteps:
+            1. Select a report that would generate a populated report on queueing.
+            Queue it, navigate to it's `Details` page, click on Configuration and
+            check the `Show Fullscreen Report` option.
+            2. Select a report that would generate an empty report on queueing.
+            Queue it, navigate to it's `Details` page, click on Configuration and
+            check the `Show Fullscreen Report` option.
+        expectedResults:
+            1. `Show Fullscreen Report` option is Enabled.
+            2. `Show Fullscreen Report` option is Disabled.
```

Comment:
**Required ** the testSteps and expectedResults should accurately reflect what is being executed in the test. So in this case, the disabled check is first and then the enabled.

Reply:
Oops, missed out on that one. Will modify it, thanks!